evcache-zipkin-tracing/src/main/java/com/netflix/evcache/EVCacheTracingTags.java

```diff
@@ -0,0 +1,15 @@
+package com.netflix.evcache;
+
+public class EVCacheTracingTags {
```

Comment:
Can we re-use tag names from `EVCacheMetricsFactory` and the IPC metrics where applicable here?

Reply:
Tag names in `EVCacheMetricsFactory` follow the camel case. All our tags are lower case strings with underscores.

Reply:
I was thinking of these:
* `evcache.call` vs `evc.call`
* `evcache.app_name` vs `ipc.server.app`
* `evcache.status` vs `ipc.status`
and possibly some of the others.

ie. prefer using existing tag names where they exist, rather than minting new ones.